install.py

```diff
@@ -145,6 +145,14 @@ def go(ui, args, answerfile_address, answerfile_script):
         elif opt == "--netinstall":
             results['netinstall'] = True
             logger.log("This is a netinstall.")
+        # XCP-ng: no-repo-gpgcheck
```

Comment:
I would remove this comment and the next one, especially if we plan to contribute this upstream (without the comments) and then merge upstream back to our fork.